src/open_inwoner/ssd/xml.py

```diff
@@ -22,7 +22,9 @@ def format_float_repr(value: str) -> str:
 
 
 def format_address(street_name: str, house_nr: str, house_letter: str) -> str:
-    return f"{street_name} {house_nr} {house_letter}"
+    if house_letter:
+        return f"{street_name} {house_nr} {house_letter}"
+    return f"{street_name} {house_nr}"
 
 
 def format_date(date_str) -> str:
```

Comment:
Looks like this PR adds a lot of glom() empty string default arguments (below), but these  `format_`and `calculate` functions don't handle the empty string values.

For example this `datetime.strptime(date_str, "%Y%m%d").strftime("%d-%m-%Y")` raises `ValueError: time data '' does not match format '%Y%m%d'` when `date_str` is empty.